src/balancerd/src/lib.rs

```diff
@@ -582,11 +600,7 @@ impl PgwireBalancer {
 
         // Now blindly shuffle bytes back and forth until closed.
         // TODO: Limit total memory use.
-        // Ignore error returns because they are not actionable, and not even useful to record
-        // metrics of. For example, running psql in a shell then exiting with ctrl+D produces an
-        // error, even though it was an intended exit by the user. Those connections should not get
-        // recorded as errors, as that's probably a misleading metric.
```

Comment:
I did not find this comment to be accurate. In fact I could not figure out a way to get this to return an error with local testing.